config/secrets.yml

```diff
@@ -24,3 +24,4 @@ production:
   secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
   omniauth_client_id: ""
   omniauth_client_secret: ""
+  fcm_secret: ""
```

Comment:
Shouldn't this be `gcm_secret`? (Google cloud messaging)

Reply:
Well, maybe, we're using Firebase Cloud Messaging, but the API is the same